portfolio/src/main/webapp/index.html

```diff
@@ -65,31 +55,30 @@ <h1>Blog</h1>
             <option value="#wellbeing">#wellbeing</option>
           </select>
           <br>
+          <input class="hidden" type="text" id="post-parentID" name="parentID" value="0"><br>
           Upload an image:
-          <input type="file" id="image" name="image">
-        </form>
-        <button onclick="sendAndGet('sender', 'text-input', 'tags')">Post</button> 
+          <input type="file" id="post-image" name="image">
+          <button type="submit">Post</button>
+        </form> 
+        
         <div id="comments-container">
           <ul id="posts-list"></ul>
         </div>
+
         <div id="confirm"></div>
-      </div>
 
-      <div id="blogreplycontent">
-        <form id="my-form" enctype="multipart/form-data">
-          <textarea id="sender-reply" rows="1" cols="20" placeholder="Nickname"></textarea> 
-          <br>
-          <textarea id="text-input-reply" rows="6" cols="60" placeholder="What's on your mind..."></textarea>
+        <form id="reply-form" enctype="multipart/form-data">
+          <textarea id="reply-text-input" name="text-input" rows="3" cols="60" placeholder=""></textarea>
           <br>
-          <div id="hidden-reply">
-            <input type="text" id="parentID" value="0"><br>
-            <input type="text" id="tags-reply" value="#general"><br>
+          <div class="hidden">
+            <input type="text" id="reply-parentID" name="parentID" value="0"><br>
+            <input type="text" id="reply-tag" name="tags" value="#general"><br>
```

Comment:
Just a thought - should this value be assigned as the Parent comment's tag? Does that already happen somewhere?

Reply:
Yeah it happens inside the handleReplies function in script.js